shared_kv_storage/Dockerfile

```diff
@@ -42,26 +41,48 @@ RUN apt-get update \
 
 RUN pip3 install --upgrade setuptools
 
-ENV TCF_HOME=/project/avalon
-
 # Make Python3 default
 RUN ln -sf /usr/bin/python3 /usr/bin/python
 
-COPY . /project/avalon/
+
+# Build Shared KV intermediate docker image.
+FROM base_image as build_image
+
+ENV TCF_HOME=/project/avalon
+
+# Copy only required files for building shared KV build image.
+COPY VERSION /project/avalon/

```

Comment:
remove "/" at the end of above COPY command

Reply:
VERSION is a file. trailing "/" is needed. Please check the COPY command in Dockerfile